doc/release-notes.md

```diff
@@ -262,6 +262,11 @@ Graphical User Interface (GUI)
   balance shown if the wallet was created using the `createwallet` RPC
   and the `disable_private_keys` parameter was set to true.
 
+- The launch-on-startup option is no longer available on macOS if
+  compiled with macosx min version greater then 10.11 (use
+  CXXFLAGS="-mmacosx-version-min=10.11" for setting the deployment
```

Comment:
`CFLAGS="-mmacosx-version-min=10.11"` could be added as well.
Without `CFLAGS` there are linker warnings:
![screenshot from 2019-01-21 11-38-04](https://user-images.githubusercontent.com/32963518/51465744-376e5b80-1d71-11e9-8363-f167505cb535.png)